frontend/licenses/licenseInfos.json

```diff
@@ -131,6 +131,15 @@
     "license": "MIT",
     "licenseText": "The MIT License (MIT)\n\nCopyright © 2018 Ruben Taelman\n\nPermission is hereby granted, free of charge, to any person obtaining a copy\nof this software and associated documentation files (the \"Software\"), to deal\nin the Software without restriction, including without limitation the rights\nto use, copy, modify, merge, publish, distribute, sublicense, and/or sell\ncopies of the Software, and to permit persons to whom the Software is\nfurnished to do so, subject to the following conditions:\n\nThe above copyright notice and this permission notice shall be included in\nall copies or substantial portions of the Software.\n\nTHE SOFTWARE IS PROVIDED \"AS IS\", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR\nIMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,\nFITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE\nAUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER\nLIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,\nOUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN\nTHE SOFTWARE.\n"
   },
+  "lamejs": {
```

Comment:
This should not be in here as the license is not compatible and we do not use audio anywhere. This one has the tendency to sneak back in again and again. Its been pulled in by the vue-advanced-chat and should be optional. I fixed this a while back in the chat repo, so either its broken again or we are pulling optional dependencies now.

Reply:
Will add a fix in the project .npmrc (optional=false). This seems to solve this issue for good and should set this option for all npm install calls